<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomsDialog</class>
 <widget class="QDialog" name="RoomsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Floor 1</string>
   </property>
   <widget class="QPushButton" name="pb101">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>101</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb102">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>102</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb103">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>103</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb104">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>104</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>561</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Floor 2</string>
   </property>
   <widget class="QPushButton" name="pb201">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>201</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb202">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>202</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb203">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>203 [♦SUITE♦]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
